### ISSUE TITLE
Integrate Mail Chimp for the newsletter.

### ISSUE THREAD

**Issue by Ryan-Koch** [Code-For-Chicago/ceic_wordpress#27]
Ask CEIC for MailChimp API key
Mailchimp has a plugin, so certain fields will be available. See step 5 in their instructions: https://mailchimp.com/help/connect-or-disconnect-list-subscribe-for-wordpress/

**Comment by egfrank** [comment]
I did a little research into this issue this afternoon. First, the Mailchimp plugin from Mailchimp seems like it has fairly bad reviews:
https://wordpress.org/plugins/mailchimp/#reviews 

This one has good reviews and is easy to use:
https://wordpress.org/plugins/mailchimp-for-wp/

I tested out the latter plugin and was able to successfully use this plugin to subscribe my email to my own personal mailchimp account. 

To set it up all we need to do is enter the Mailchimp API Key. Then, we can set what list we're sending it to using the settings for the plug-in. If CEIC has multiple email campaigns we'll have to ask them which one they want to connect this list-subscribe to, but it's possible that once we add the API key we see they've only set up one option called "Newletter" or the like. There are also options to add fields like "last name" "first name" etc, if that's interesting to CEIC.  

Let me know our thoughts on using the 3rd party plugin as opposed to the Mailchimp official one are!
